components/org.wso2.carbon.identity.workflow.engine/src/main/java/org/wso2/carbon/identity/workflow/engine/util/WorkflowEngineConstants.java

```diff
@@ -166,30 +171,40 @@ public static class ParameterValue {
      */
     public enum ErrorMessages {
 
-        TASK_NOT_FOUND("SWE_00001", "Invalid event ID"),
-        WORKFLOW_ID_NOT_FOUND("SWE_00003", "The workflow Id is not valid"),
-        ERROR_OCCURRED_WHILE_RETRIEVING_WORKFLOW_REQUEST("SWE_00004",
+        /*
+         Error messages for client errors - 400 series
+         */
+        TASK_NOT_FOUND("WFE_40001", "Invalid event ID"),
+        WORKFLOW_ID_NOT_FOUND("WFE_40002", "The workflow Id is not valid"),
+        USER_ERROR_NON_EXISTING_TASK_ID("WFE_40003", "Task does not exist."),
+        USER_ERROR_NOT_ACCEPTABLE_INPUT_FOR_NEXT_STATE("WFE_40004", "Unacceptable input provided, " +
+                "Only [CLAIM, RELEASE, APPROVED, REJECTED] are acceptable."),
```

Comment:
```suggestion
        USER_ERROR_NOT_ACCEPTABLE_INPUT_FOR_NEXT_STATE("WFE_40004", "Unacceptable input provided, " +
                "Allowed values: CLAIM, RELEASE, APPROVED, REJECTED."),
```

can we modify error message to get rid of the array? WDYT?